elixir/apps/web/lib/web/components/core_components.ex

```diff
@@ -221,7 +190,7 @@ defmodule Web.CoreComponents do
     <.paginator
       page={5}
       total_pages={100}
-      collection_base_path={~p"/users"}/>
+      collection_base_path={~p"/actors"}/>
```

Comment:
Yeah, definitely agree this is a tough naming problem.

Will service/machine accounts be listed along side human users?  Or will they have their own URL path?

Reply:
I was planning to have them in a separate area in the UI, so it probably makes sense to scope the routes then?

Reply:
Well, the reason I was asking was that I had the thought that if everything is under one page, then having the URL have `actors` in it might not be a big deal, but if things are going to get split out with having a page of only human users then it may be better to have the URL say `users`? 🤷‍♂️

Reply:
@bmanifold I was thinking the same thing. `users` and `service_accounts` may be the same thing in the backend, but from an Admin's perspective, they're really two different things with different security contexts and workflows.

Reply:
Hmm, but this makes for more UIs for us to build. I suppose we could keep things simpler for now, and just have one `actors` UI section with controls for segregating `service_account` actors and `user` actors.

On that note, we may also want to update `device` to `client` like @AndrewDryga had mentioned; I agree that `Device` is not a great name for headless clients running via `service_accounts`.

Reply:
We could discuss more on Slack or a quick call